backend/Origam.Server/Resources/SharedResources.cs-CZ.resx

```diff
@@ -417,4 +417,7 @@
   <data name="TwoFactorCodeRequired" xml:space="preserve">
     <value>Kód pro dvoufaktorové přihlášení musí být vyplněn</value>
   </data>
+  <data name="Reset" xml:space="preserve">
+    <value>Obnovit</value>
```

Comment:
Where is this used? I would expect the text to be a bit longer. Something like "Reset your password" or "Did you forget your password?".

Reply:
Then I suggest to make the text longer, as is usual today on the websites. I think the most proper is "Did you forget your password?".

Reply:
<img width="428" alt="Screenshot 2022-03-02 at 11 46 49" src="https://user-images.githubusercontent.com/83349812/156347459-e4ef943f-5b8d-46ac-a0f9-67f26de5a8fb.png">